client/payment-details/summary/index.js

```diff
@@ -117,20 +129,16 @@ const PaymentDetailsSummary = ( { charge = {}, isLoading } ) => {
 				<div className="payment-details-summary__section">
 					{ /* TODO: implement control buttons depending on the transaction status */ }
 					<div className="payment-details-summary__actions">
```

Comment:
Changed it straight to `payment-details-summary__id` in https://github.com/Automattic/woocommerce-payments/pull/932/commits/337550ed9c4d5463848a3fc3796a8661aed76a60 (not sure how that extra prettier change ended up in that commit instead of an earlier one).